test/integration/deploy/channels/slack/test.verify-slack.js

```diff
@@ -112,4 +150,13 @@ describe('deploy verify-slack integration tests', () => {
       .update('authorize')
       .digest('hex');
   }
+
+  function validatePipelineCreation(pipelineName) {
```

Comment:
shouldn't the name of this method be a little more generalized? this method basically checks if an _action_ exists in the user deploy namespace, and it doesn't really need to have to do anything with pipelines.